app/forms/validators.py

```diff
@@ -128,39 +128,64 @@ def __call__(
         field: Union[DecimalFieldWithSeparator, IntegerFieldWithSeparator],
     ) -> None:
         value: Union[int, Decimal] = field.data
+        decimal_places = 0
+        if field.type == "DecimalFieldWithSeparator":
```

Comment:
Check using `isinstance` instead of the literal string type.

Reply:
isinstance evaluates as False for field.type as DecimalFieldWithSeparator (evaluates True for str)

Reply:
It shouldn't be `isinstance` on `field.type`, it should be `isinstance` on field itself.

Reply:
Updated 👍